src/components/BookCard.js

```diff
@@ -13,28 +13,29 @@ const BookCard = ({ books }) => {
       books.map((book) => (
         <div key={book.item_id} className="bookCard">
           <div className="book">
-            <p>{book.category}</p>
-            <h1>{book.title}</h1>
-            <p>{book.author}</p>
+            <p className="bookCategory">{book.category}</p>
+            <h1 className="bookTitle">{book.title}</h1>
+            <p className="bookAuthor">{book.author}</p>
             <div className="Book-btns">
-              <button type="button">Comments</button>
-              <div className="divide" />
-              <button onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
-              <div className="divide" />
-              <button type="button">Edit</button>
+              <button className="comment" type="button">Comments</button>
+              <div className="Line-2" />
+              <button className="remove" onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
+              <div className="Line-2" />
+              <button className="edit" type="button">Edit</button>
             </div>
           </div>
           <div className="bookInfo">
             <div className="progress" />
             <div className="progressInfo">
-              <p>
+              <p className="percentage">
                 64%
               </p>
-              <p>Completed</p>
+              <p className="completed">Completed</p>
             </div>
+            <div className="Line-3" />
```

Comment:
- [ ] As I mentioned above :point_up: please stick to this Zaplin design if you check dividers between the percentage and progress button the width is not matched with Zaplin Design so kindly fix this issue as well 🙂

**Your Design**
![Screenshot from 2023-08-24 00-51-25](https://github.com/As1imwe-Mark/book_store/assets/45226848/17c37779-787f-4203-abb2-bcb2d2bf44e2)

**Zaplin Design**
![Screenshot from 2023-08-24 00-51-35](https://github.com/As1imwe-Mark/book_store/assets/45226848/bb40ab13-e059-4441-a8ea-ef0758153696)